.github/workflows/issue_commands.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+      "type": "label",
+      "name": "type/docs",
```

Comment:
I'd guess this wont work since we don't have that label, we have `documentation`

Reply:
Yeah I don't think it'll work either. We need to add the label and probably best to then delete the `documentation` label.

Reply:
I believe it would be better if there is a convention in the docs team to use `type/docs` across repos.

Reply:
done